Sources/NetworkProtection/PacketTunnelProvider.swift

```diff
@@ -263,10 +263,20 @@ open class PacketTunnelProvider: NEPacketTunnelProvider {
             providerEvents.fire(.rekeyAttempt(.success))
         } catch {
             providerEvents.fire(.rekeyAttempt(.failure(error)))
+            await subscriptionAccessErrorHandler(error)
```

Comment:
User access errors are handled at the topmost level, instead of within `updateTunnel`.  This is because not all calls to `updateTunnel` need the tunnel to shut down itself.

In fact when starting up the tunnel we want to fail startup returning the relevant error instead.